web_scraper/main.py

```diff
@@ -122,6 +124,23 @@ def execute(target_url):
             
         logger.info(f"Crawl complete. Total pages scraped: {len(crawl_results)}")
 
+        email_writer_path = site_dir / 'email.txt'
+        if content_file and Path(content_file).exists():
+            # Step 3: Generate email using EmailWriter
+            logger.info("Step 3: Generating email from aggregated content...")
+            # Add parent directory to path to import email_writer from root
+            sys.path.insert(0, str(Path(__file__).parent.parent))
+            from email_writer import EmailWriter  # Import here to avoid circular imports
```

Comment:
circular imports are annoying, but we can typically find a way to avoid them and we should!

we'll try to fix it